x-pack/solutions/security/packages/kbn-cloud-security-posture/public/src/components/fleet_extensions/hooks/use_cloud_setup_context.tsx

```diff
@@ -89,18 +89,6 @@ const isCloudConnectorEnabledForProvider = ({
   const providerConfig = config.providers[provider];
   const cloudConnectorEnabledVersion = providerConfig.cloudConnectorEnabledVersion;
 
-  const hostProvider = getCloudProviderFromCloudHost(cloud?.cloudHost);
-
-  // Cloud connector availability rules:
-  // - AWS: Only available on AWS host
-  // - GCP: Not enabled yet (always disabled)
-  // - Azure: Available on any cloud host
-  if (!hostProvider) return false;
-
-  if (provider === AWS_PROVIDER && hostProvider !== AWS_PROVIDER) return false;
-  if (provider === GCP_PROVIDER) return false; // GCP cloud connectors not enabled yet
```

Comment:
It is true that we don't care anymore about the `hostProvider`, however, you can't remove this row since we still don't support GCP cloud connectors.
This PR should enable AWS cloud connectors when kibana running on GCP/Azure.

In general, I think we should have an allowlist - so something like:
```
if provider not in [AWS_PROVIDER, AZURE_PROVIDER]
  return false
```